README.md

```diff
@@ -487,16 +531,16 @@ Add `/etc/hosts` entry to match hostname.
    127.0.0.1 ballerina.guides.io
 ```
 
-Access the service 
+Invoke the service
 ```bash 
-   curl -v -X POST -d \
+   $ curl -v -X POST -d \
    '{ "Order": { "ID": "100500", "Name": "XYZ", "Description": "Sample order."}}' \
    "http://ballerina.guides.io/ordermgt/order" -H "Content-Type:application/json" 
 ```
 
 ## Observability 
-Ballerina is by default observable. Meaning you can easily observe your services, resources, etc.
-However, observability is disabled by default via configuration. Observability can be enabled by adding following configurations to `ballerina.conf` file and starting the ballerina service using it. A sample configuration file can be found in `restful-service/guide/restful_service`.
+Ballerina comes with support for observability built-in to the language.
+Observability is disabled by default. It can be enabled by adding the following configurations to `ballerina.conf` file in `restful-service/guide/`. A sample configuration file can be found in `restful-service/guide/restful_service`.
```

Comment:
Minor corrections:
"Observability is disabled by default. 
To enable Observability, add the following configurations to the `ballerina.conf` file in `restful-service/guide/`. A sample configuration file can be found in `restful-service/guide/restful_service`."